internal/report/report_test.go

```diff
@@ -235,6 +235,46 @@ func TestNewPolicyReportResult(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "Validating policy, allowed response with no message",
```

Comment:
Instead adding a new test we should modify existing tests in this file and the mock policy server handler in `scanner_test.go` to remove the Result when the `AdmissionReview` is allowed

Reply:
Yeah, I was thinking about that too. But I've decided to keep the old tests just in case I was missing something that you know. As you are not around and leave that for the review process, like know. :)

I've just updated the tests. Please, give another look.